dapps-certification-persistence/src/IOHK/Certification/Persistence/API.hs

```diff
@@ -127,8 +128,8 @@ syncRun runId time= update runs
     (\run -> run ! #runId .== literal runId)
     (`with` [ #syncedAt := literal time ])
 
-createCertificate :: (MonadSelda m,MonadMask m) => UUID -> Text -> UTCTime -> m (Maybe Certification)
-createCertificate runId ipfsCID time = transaction $ do
+createCertificate :: (MonadSelda m,MonadMask m) => UUID -> Text -> Text -> UTCTime -> m (Maybe Certification)
```

Comment:
Can we improve this type signature somehow? `Text` doesn't tell me much what should go here, should we have a domain-specific newtype wrapper or perhaps a record of named arguments?